sdk/storage/storage-file-datalake/src/utils/utils.common.ts

```diff
@@ -633,3 +633,31 @@ export function EscapePath(pathName: string): string {
   }
   return split.join("/");
 }
+
+export interface RawResponseWithEncryptionContextLike {
+  encryptionContext?: string;
+  _response: HttpResponse & {
+    parsedHeaders: {
+      encryptionContext?: string;
+    };
+  };
+}
+
+/**
+ * Parse value of encryption context from headers in raw response.
+ */
+export function ParseEncryptionContextHeaderValue(
```

Comment:
It looks not good to parse raw responds, it will impact code stability and not easy to rmaintain.
Is there any better way?

Besides that, we might can write a generic function to parse a header with specific name.  instead of a function can only parse Encryption Context.

Besides